dbt/adapters/athena/impl.py

```diff
@@ -156,7 +156,7 @@ def add_lf_tags_to_database(self, relation: AthenaRelation) -> None:
             LOGGER.debug(f"Lakeformation is disabled for {relation}")
 
     @available
-    def add_lf_tags(self, relation: AthenaRelation, lf_tags_config: Dict[str, Any]) -> None:
+    def add_lf_tags(self, relation: AthenaRelation, lf_tags_config: dict[str, Any]) -> None:
         config = LfTagsConfig(**lf_tags_config)
```

Comment:
here we have `lf_client` and we use other aws clients through the code. WDYT is it possible to move them to `cached_property` and just reuse (finally `cached_property` is available in python3.8)

Reply:
I think this should be part of an other PR because there are a lot of clients everywhere (athena, s3, glue, lf) and it depends also on what we decide for #443